README.md

```diff
@@ -39,34 +33,33 @@ Below features are under consideration to be included in the future releases
 
 ## Hardware Requirements
 
-### Intel® SGX supported platform
+### Intel SGX supported platform
 
 * Third Generation Intel® Xeon® Scalable Processors (or later version) are used by the cluster.
 * Contact your server or BIOS vendor for the BIOS setting to enable the feature.
 
-### Intel® dGPU Card supported platform
+### Intel Data Center GPU card supported platform
 
 * The Intel® Data Center GPU Flex Series 140 or Intel® Data Center GPU Flex Series 170 Card is enabled on the nodes.
 * Contact your server or BIOS vendor for the BIOS setting to enable the cards.
 
-### Intel® QAT supported platform
+### Intel QAT supported platform
 * 4th Gen Intel® Xeon® Scalable Processors (or later versions) are used by the cluster.
 * Contact your server or BIOS vendor for the BIOS setting to enable the feature.
 
 ## Get Started 
-To properly provision Intel Hardware features, deploy and manage the related e2e solutions as well as the reference workload, below OCP software stack development and life cycle management flow is followed by this project
+To properly provision Intel hardware features, deploy and manage the related e2e solutions as well as the reference workload, below OCP software stack development and life cycle management flow is followed by this project
 
 ### Day 0 - Define the requirements of the OCP platform and design it.
 Red Hat [OpenShift Operator](https://www.redhat.com/en/technologies/cloud-computing/openshift/what-are-openshift-operators) automates the creation, configuration, and management of instances of Kubernetes-native applications. It is based on [Kubernetes operator pattern](https://kubernetes.io/docs/concepts/extend-kubernetes/operator/). So the software stack running on OCP needs to be containerized and managed by a specific Operator. As an essential part of OCP, operators need to be well-designed from Day 0. Good examples are [Intel Device Plugins Operator](https://github.com/intel/intel-device-plugins-for-kubernetes) and [KMM Operator](https://github.com/rh-ecosystem-edge/kernel-module-management).
 
 ### Day 1 - provision the OCP platform and configure it to a working state.
-This project mainly focuses on bare metal OCP cluster. [DCI](https://doc.distributed-ci.io/dci-openshift-agent/) is used to provision the bare metal OCP cluster. Users can also refer to [bare metal OCP cluster installation instructions](https://docs.openshift.com/container-platform/4.11/installing/installing_bare_metal_ipi/ipi-install-overview.html) to install the bare metal OCP cluster.
+This project mainly focuses on bare metal OCP cluster.  [Distributed CI (DCI)](https://doc.distributed-ci.io/dci-openshift-agent/) is used to provision the bare metal OCP cluster. Users can also refer to [bare metal OCP cluster installation instructions](https://docs.openshift.com/container-platform/4.11/installing/installing_bare_metal_ipi/ipi-install-overview.html) to install the bare metal OCP cluster.
```

Comment:
Can we update link to point to `4.12` installation instructions?

Reply:
If there are any other issues, I think we can merge this PR, and you can submit a PR to fix the link issue. Is that OK?